### ISSUE TITLE
Unstable behavior in ping-pong application

### ISSUE THREAD

**Issue by Suhoy95** [silent-dragonfly/cc3100-L2-communication#1]
Periodically device are stopping and receiving fails:

![capture](https://user-images.githubusercontent.com/6908059/50452784-11c87080-08f1-11e9-8c9e-6caf95caab3b.PNG)

Before sending to the ti.com forum, it requires to ensure:
- the version of CC3100 service pack
- version of driver sources
- version of SimpleLink Studio port

**Comment by Suhoy95** [comment]
Maybe, problem in default connection policies. Try [this one](https://github.com/silent-dragonfly/cc3100-wireshark-sniffer/blob/master/src/main.c#L234) and [this](https://github.com/silent-dragonfly/cc3100-wireshark-sniffer/blob/master/src/main.c#L308).

**Comment by jluisftapia** [comment]
![capture](https://user-images.githubusercontent.com/44317733/50521155-259ede80-0a79-11e9-87ed-389d6b9d5626.PNG)

**Comment by Suhoy95** [comment]
Yes, the recommendation from [User Guide](http://www.ti.com/lit/ug/swru368b/swru368b.pdf), section 16.7 helps. But There is strange SL_SOCK_TX_FAILED_EVENT from in ponger.
